examples/objective-c-ios/Bugsnag Test App/AppDelegate.m

```diff
@@ -13,9 +13,51 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     
+    /**
+     This is the minimum amount of setup required for Bugsnag to work.  Simply input your apiKey and the application will deliver all error and session notifications to the appropriate dashboard.
+     */
     NSString *apiKey = @"<YOUR_APIKEY_HERE>";
-    BugsnagConfiguration *config = [[BugsnagConfiguration alloc] initWithApiKey:apiKey];
-    [Bugsnag startWithConfiguration:config];
+    [Bugsnag startWithApiKey:apiKey];
+    
+    /**
```

Comment:
I wonder if we are making a rod for our own backs here by duplicating information that is available in the docs.  I personally find that commented out code can be distracting and would rather copy/paste it for myself from the docs as required.  It's also one less thing for us to maintain should config properties change over time.